File: src/main/resources/static/src/app/issuing-module/shared/model/configuration/variable-data-option.ts
import { JsonMember, JsonObject } from 'typedjson-npm/src/typed-json';

import { ISelectable } from '../../../../shared/dyna-field/selectable';

/**
* this object represents a variable data option 
*
* @author jtorres
*/
@JsonObject
export class VariableDataOption implements ISelectable {
    @JsonMember
    id: number;
    @JsonMember
    code: string;
    @JsonMember
    description: string;
}

File: src/main/java/co/com/fsighttech/imp/issuing/bo/configuration/VariableDataOption.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo.configuration;

import lombok.Getter;
import lombok.Setter;

/**
 * Represents an  option for a variable data.
 * 
 * @author jtorres
 *
 */

@Getter
@Setter
public class VariableDataOption {
	
	private Long 	id;
	private String 	code;
	private String  description;
}